common/src/main/java/us/ilite/common/lib/control/PIDGains.java

```diff
@@ -0,0 +1,19 @@
+package us.ilite.common.lib.control;
+
+public class PIDGains {
+    public final double mP;
+    public final double mI;
+    public final double mD;
+    public final double mF;
+
+    public PIDGains( double kP, double kI, double kD ) {
```

Comment:
Since these are parameters, these should be prefixed with `p` instead of `k` (which is for constants). See https://github.com/iliterobotics/FRC-Robot-2019#conventions.